docs/fundamentals/documentarray/interaction-cloud.md

```diff
@@ -0,0 +1,41 @@
+(interaction-cloud)=
```

Comment:
This is duplicate
We already added a cloud support section in this PR
https://github.com/docarray/docarray/pull/697